index.html

```diff
@@ -1,183 +1,376 @@
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button class="resume">Get My Resume</button>
+        <hr />
+      </div>
+      <div class="work-on">
+        <h3 class="best">
+          <img src="icons/Rectangle55.png" alt="rectangl" /> Language
+        </h3>
+        <ul class="list">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <h3 class="best">
+          <img src="icons/Rectangle56.png" alt="rectangle" /> Frameworks
+        </h3>
+        <ul class="list">
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>Rspec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <h3 class="best">
+          <img src="icons/Rectangle57.png" alt="circle" /> Skills
+        </h3>
+        <ul class="list">
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="myresume">
+        <h3 class="h3-title">About me</h3>
+        <p class="about-paragraph">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button class="resume">Get My Resume</button>
+        <hr />
+      </div>
+      <div class="work-on">
+        <h3 class="best">
+          <img src="icons/Rectangle55.png" alt="rectangl" /> Language
+        </h3>
+        <ul class="list">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <h3 class="best">
+          <img src="icons/Rectangle56.png" alt="rectangle" /> Frameworks
+        </h3>
+        <ul class="list">
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>Rspec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <h3 class="best">
+          <img src="icons/Rectangle57.png" alt="circle" /> Skills
+        </h3>
+        <ul class="list">
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <section class="contact-form">
+      <div class="the-contact">
+        <p class="contact-p">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+
+        <form action="https://formspree.io/f/" method="post">
```

Comment:
- [ ] [OPTIONAL] you have done a great job implementing some of the recommendations of the previous reviewer, however, you still need to stick to the design as much as possible.  You have omitted the last part of the form that has the list of social media icons. Also, fix the margin-top between the Get in touch `button` and the text area.
 
Your Implementation             |  Figma Design
:-------------------------:|:-------------------------:
![your-design](https://user-images.githubusercontent.com/58692590/149210163-a1ccbac9-9764-4c18-aab4-6fa5de0f9d11.png) | ![figma2](https://user-images.githubusercontent.com/58692590/149210126-ea444031-303e-41a4-b34e-5ffac8e413b1.png)